### DISCUSSION TITLE
Should we add MRIcroGL and BrainNetViewer?

### DISCUSSION THREAD

**Discussion by civier** [neurodesk/neurodesk.github.io-hugo#17] [Q&A] [answered]
We consider to add these packages at one point. Can people say if they need them as well?
However, both of them probably require GPU support in VNM, as don't work as-is when manually installed
(Isabella Beukelaar request and report)

**Comment by stebo85** [comment, marked as answer]
yes, I think it's a good idea. A good place to start would be here: https://github.com/NeuroDesk/caid/ - start from one example and send a pull request.
